code/API_definitions/Edge-Application-Management.yaml

```diff
@@ -908,6 +912,8 @@ components:
           $ref: '#/components/schemas/EdgeCloudProvider'
         edgeCloudRegion:
           $ref: '#/components/schemas/EdgeCloudRegion'
+        edgeCloudKubernetesClusterRef:
+          $ref: '#/components/schemas/KubernetesClusterRef'
```

Comment:
Sorry, just noticed this, but I don't think the kubernetes cluster ref belongs here. It's not really a property of an EdgeCloudZone. I think we need a separate object to be able to specify the optional cluster ref as part of the create appinstance API that can be arrayed. Also discussion #256 is relevant here.

Reply:
Hi @gainsley, I've added for the GET /apps/{appId}/instances to retrieve the cluster where an application is deployed. I've noticed that it also impacts on the POST where it is only specified the edgeCloudZone. What do you think if we put the k8s cluster ref outsie from the edgeCloudZone but inside of appInstanceInfo?.

Reply:
Yes, I think it makes sense to have the clusterref in appInstanceInfo. Unfortunately though, that doesn't help the `post /app/{appId}/instances` case since that doesn't use `appInstanceInfo`. There I think you'll need to create a new wrapper object that is { clusterRef (optional), EdgeCloudZoneId } that can then be arrayed.